### PR TITLE
Fix connector details in kafka-connect-cluster.json

### PR DESCRIPTION
**Pull request by tsuz**
Fix connector details not showing. It requires table migration due to an old angular plugin, according to Grafana's recommendation. After migrating it, the widget shows properly.

| Before | After |
|--|--|
| Screenshot 2023-10-23 at 19 21 31  
Screenshot 2023-10-23 at 19 21 38 | Screenshot 2023-10-23 at 19 23 35 |

### CODE REVIEW

**Review by hifly81 — CHANGES_REQUESTED** [review]
Hi @tsuz I see many changes in the dashboard and it's hard to follow what you have really updated.
could you please verify?

**Comment by tsuz** [comment]
@hifly81 Because the widget has changed, it's extremely difficult to follow using JSON. This is the new dashboard created based on the JSON from this branch. 

https://github.com/confluentinc/jmx-monitoring-stacks/assets/6927131/1258d59b-3841-4da0-b414-de37fc40134e

**Review by hifly81 — APPROVED** [review]
LGTM!
